db/schema.rb

```diff
@@ -25,12 +25,22 @@
     t.index ["user_id"], name: "index_comments_on_user_id"
   end
 
+  create_table "countries", force: :cascade do |t|
+    t.string "code"
+    t.float "lat"
+    t.float "lng"
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
   create_table "locations", force: :cascade do |t|
-    t.string "country"
```

Comment:
@iandouglas I just finished changing country from a string to its own model, and have everything working in development. Now I want to deploy, but my production db already has entries and I am not sure how to handle converting them. I also want to seed the production database with all the countries and am not sure how to go about it. Any advice?